Mongle/Projects/Features/ChatFeature/Sources/ChatView.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import SwiftUI
+import Core
```

Comment:
```suggestion
import Core
import SwiftUI
```

import문은 알파벳 순으로 정렬해주세요

Reply:
suggestion commit 감사합니다! 반영하게요